setup.py

```diff
@@ -1,5 +1,17 @@
 # borgbackup - main setup code (see also other setup_*.py files)
 
+# NOTE(pabelanger): This tries to workaround the chicken/egg issue where we
+# now need pkgconfig / Cython installed before we can properly compile our
+# setup.py script.
+from setuptools.command import easy_install
+easy_install.main(['pkgconfig'])
+easy_install.main(['Cython'])
+
+# We need to reload setup.py to ensure modules above are not in PYTHONPATH.
+import importlib
+import site
+importlib.reload(site)
```

Comment:
and why is it needed?

Reply:
This was my fix to import pkgconfig not working, because I assumed it was no in the pythonpath when pip install ran.  I'm open to other suggestions, but removing it will still results in the failure above.